src/test/java/com/cydeo/runner/CukesRunner.java

```diff
@@ -15,7 +15,7 @@
         features = "src/test/resources/features" ,
         glue = "com/cydeo/steps",
         dryRun = false,
-        tags = "@smoke"
```

Comment:
Hey Soelma everything is good ,just put it back tag in cukes runner to @smoke instead of @addbook, that part usually should be empty but than left @smoke so if you delete and add yours it will be conflict because master line ,framework will be changed , so just remove that and put back to @smoke ,and i will approve it :)

Reply:
Thank you @loshmi04, will do.